src/integration-test/java/com/mongodb/jdbc/integration/testharness/IntegrationTestUtils.java

```diff
@@ -209,117 +209,133 @@ private static void validateResultSetMetadata(TestEntry test, ResultSetMetaData
             throws SQLException, IllegalAccessException {
         int columnCount = rsMetaData.getColumnCount();
         if (test.expected_sql_type != null) {
-            assertEquals(test.expected_sql_type.size(), columnCount);
+            assertEquals(test.expected_sql_type.size(), columnCount, "Either the yml test specification is missing entries for 'expected_sql_type' or the data in the DB is not matching the test");
             for (int i = 0; i < columnCount; i++) {
                 int sqlType = TestTypeInfo.typesStringToInt(test.expected_sql_type.get(i));
-                assertEquals(sqlType, rsMetaData.getColumnType(i + 1));
+                assertEquals(sqlType, rsMetaData.getColumnType(i + 1), "Invalid getColumnType result for column " + i+1);
             }
         }
         if (test.expected_catalog_name != null) {
-            assertEquals(test.expected_catalog_name.size(), columnCount);
+            assertEquals(test.expected_catalog_name.size(), columnCount, "Either the yml test specification is missing entries for 'expected_catalog_name' or the data in the DB is not matching the test");
             for (int i = 0; i < columnCount; i++) {
-                assertEquals(test.expected_catalog_name.get(i), rsMetaData.getCatalogName(i + 1));
+                assertEquals(test.expected_catalog_name.get(i), rsMetaData.getCatalogName(i + 1),"Invalid getCatalogName result for column " + i+1);
```

Comment:
```suggestion
                assertEquals(test.expected_catalog_name.get(i), rsMetaData.getCatalogName(i + 1), "Invalid getCatalogName result for column " + i+1);
```
I really appreciate all of these test messages -- great idea! [nit] on this one: spacing

Reply:
Splotless moved it down the next line. Not an issue anymore :)